Anytype/Sources/PresentationLayer/TextEditor/EditorPage/Views/Settings/Icon/Modules/ObjectIconPickerModuleAssembly.swift

```diff
@@ -3,16 +3,11 @@ import UIKit
 import SwiftUI
 
 protocol ObjectIconPickerModuleAssemblyProtocol {
-    func make(
-        document: BaseDocumentGeneralProtocol,
-        onIconAction: @escaping (ObjectIconPickerAction) -> Void
```

Comment:
Deleted - `onIconAction: @escaping (ObjectIconPickerAction) -> Void`
Now this screen contains icon uploading logic. No needed that this logic will be implemented in header view model